101-media-services-create/azuredeploy.json

```diff
@@ -0,0 +1,57 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "name": {
```

Comment:
@mathieu-benoit looks like mediaservicename has to be unique, is that across the region or across everything?